### DISCUSSION TITLE
Discord integration: Server status cards and chat relay support forum posts and threads

### DISCUSSION THREAD

**Discussion by MarkimusDev** [gameserverapp/Platform#413] [Suggestions]
I am suggesting that the discord integration elements of GSA are updated to include support for several new features discord implemented a while back.

When telling GSA where to send info to discord, allow us to select from:
- Channel (Current implementation)
- Forum post (Provide thread ID and the server card or chat is sent there)
- Threads (As above but with a thread id)

This would allow us to use these new features to organise large amounts of servers better.

I will use my setup as an example. Right now I have about 19 servers spanning across 3 main ark mods. My discord setup is as follows:

- #ark-info (Giant text wall of info about my servers as a whole, and each individual cluster with links to mod lists, info about rates, etc)
- #ark-serverstatus (all server status cards go here, unorganised)
- #ark-chat-omega (Omega cluster chat goes here)
- #ark-chat-pf (Primal fear cluster chat)
- #ark-chat-dox (DOX cluster chat)

With the option to use forums/threads I would set it up as follows:

- #ark-servers (Forum)
  - Omega Servers (Forum post)
    - Main post containing server info, mod list, rates, etc
    - Server Status (Thread where the server status cards for each server in the omega cluster are sent)
    - Server Chat (Thread where cluster chat is sent)
  - Primal Fear Servers (Forum post)
    - Main post containing server info, mod list, rates, etc
    - Server Status (Thread where the server status cards for each server in the primal cluster are sent)
    - Server Chat (Thread where cluster chat is sent)
  - DOX Servers (Forum post)
    - Main post containing server info, mod list, rates, etc
    - Server Status (Thread where the server status cards for each server in the dox cluster are sent)
    - Server Chat (Thread where cluster chat is sent)

I feel like this would make the user experience finding info on servers they want to join a lot cleaner and more efficient.

Thanks for your consideration.

**Comment by LimitlessDrag0n** [comment]
Just as mine but better described. Think I will withdraw mine :D 

Upvoted
